onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -701,6 +701,8 @@ def _where_input_wrangler(
     "index_select": core_ops.aten_index_select,
     "layer_norm": core_ops.aten_layer_norm,
     "max": core_ops.aten_max,
+    "max_pool2d": nn_ops.aten_max_pool2d,  # Custom from extra_opinfo
```

Comment:
We've already had nn.functional.max_pool2d and nn.functional.max_pool3d tests. If we want to use this custom testing to replace them, let's remove those 2 ones from the test file.

Reply:
The new tests just test for when stride is empty